### ISSUE TITLE
icingadb feature not available

### ISSUE THREAD

**Issue by riban-bw** [Icinga/icinga2#7932]
## Describe the bug

Following the [icinga2 installation instructions ](https://icinga.com/docs/icinga2/latest/doc/02-installation/)and [icingadb installation instructions](https://icinga.com/docs/icingadb/latest/doc/02-Installation/), icingadb feature is not available.

## To Reproduce

1. Follow icinga2 installation instructions at https://icinga.com/docs/icinga2/latest/doc/02-installation
2. Follow icingadb installation instructions at https://icinga.com/docs/icingadb/latest/doc/02-Installation
3. Execute `icinga2 feature enable icingadb`

## Expected behaviour

icingadb is enabled.

## Actual behaviour

Error message:
```
critical/cli: Cannot enable feature 'icingadb'. Source file '/etc/icinga2/features-available/icingadb.conf' does not exist.
critical/cli: Cannot enable feature(s): icingadb
```
## Your Environment

Ubuntu 18.04.4 LTS Bionic 64-bit running within VirtualBox VM as Intel 64-bit architecture on 64-bit Intel host.

* Version used (`icinga2 --version`): r2.8.1-1
* Operating System and version: Ubuntu 18.04.4 LTS Bionic 64-bit 
* Enabled features (`icinga2 feature list`): checker mainlog notification

## Additional context

Fresh install of Ubuntu minimal + OpenSSH server followed by installation of icinga2 following instructions from https://icinga.com/docs/icinga2/latest/doc/02-installation.

**Comment by riban-bw** [comment]
I just repeated installation steps and now it works. The main difference here is that second run would be with testing repository enabled and I noticed that icinga2 install was er-triggered which I would not expect. I will repeat with a clean OS to ensure this wasn't finger trouble...

**Comment by riban-bw** [comment]
I just repeated and found I had to install icinga2 and plugins from main then install icingadb from testing at which point icingadb-redis failed to install due to dependencies so I had to run the install of icinga2 from main again at which point icingadb feature was available. I am not sure what state the install is in because of this iteration... There is definitely an issue with this which needs resolving.

**Comment by N-o-X** [comment]
Hi @riban-bw, thanks for reporting! Have you tried just enabling both repositories at the same time? That should resolve you issues.

**Comment by riban-bw** [comment]
I have both repros enabled and then need to redo previously performed actions. Following the installation instructions chronologically fails. It seems that you need to enable the testing repro if you want to use icingadb and that you need icinga2 (and other packages) from testing for this to work. The installation instructions should make this clear. Maybe this is temporary until icingadb moves from testing to main but in the interim - if we want anyone to have a chance of using this (and testing it) then we need instructions that allow installation. I am still struggling to get icingadb working as it does not seem to be supported by the icingaweb2 setup or some similar issue with database schema.
I need to get the testbed configured to allow me to investigate significant issues with our production system and to help improve the quality of my issue reports (and hopefully improve the icinga product) but am rather stuck at the moment!
You can see my full installation progress in issue #7934.

For now I would recommend the installation instructions for icingadb include `apt-get install icinga2` with a comment that it needs to be repeated to make it obvious to users that this step is required despite they have already performed it.

**Comment by N-o-X** [comment]
You're right, that's unfortunate. We really need to add a note to the installation section, that you need to install Icinga 2 from the testing repository, if you want to use Icinga DB. Thanks!

**Comment by nomad-cyanide** [comment]
I am new to this Icinga. Was recommended it by a friend, so I tried installing it just now and the install failed just like described here.
Furthermore, it is not really clear from the instruction, if you need Icinga DB or not, in order to use the product.
Second, I tried reading ahead to see if I could read whether I needed to install Icinga DB, in order to use Icings Web 2, but under the heading "Installing Icinga Web 2" on https://icinga.com/docs/icinga2/latest/doc/02-installation/ there is a link to "Icinga Web 2 documentation" for furhter instructions on how to install it. The link leads here: https://icinga.com/docs/icingaweb2/latest/doc/ which just gives a 
"Forbidden
You don't have permission to access this resource."
message.
Sorry to say, but Zabbix is looking better and better.

**Comment by riban-bw** [comment]
Those issues are reported and being resolved. It is a shame that the docs can be confusing. Database is required to monitor the status so yes, database installation is required to use icinga2 and icingaweb2. Hopefully this feedback may improve documentation for noobs.

**Comment by aDr1anGM** [comment]
Hi, so how would you recommend me to move forward with testing icinga2 on centos8? can you point me to any walkthrough on how to test the latest available edition for both app and web? I also ran into the same issue described before when following this tuto for Centos 8: https://icinga.com/docs/icinga2/latest/doc/02-installation/
Thank you for any help in advance.

**Comment by nomad-cyanide** [comment]
Hi again
I just tried the solution offered in issue #7934 and it seemed to do the trick to just do this:

dnf install icinga2

Then the enable command worked:

icinga2 feature enable icingadb
Enabling feature icingadb. Make sure to restart Icinga 2 for these changes to take effect.

**Comment by N-o-X** [comment]
This issue was fixed in the docs a while ago by removing Icinga DB as recommended until there's a full release. The Icinga DB feature for Icinga 2 was also released for testing with Icinga 2.12.0.
